keymaster/ta/include/master_crypto.h

```diff
@@ -22,6 +22,8 @@
 #define KEY_LENGTH 16
 #define TAG_SIZE 128U
```

Comment:
If you started doing this - could you please change the behavior of all these values - KEY_SIZE and TAG_SIZE are used very rarely, and this is quite confusing. As for me - it will be better to operate with KEY_LENGTH, TAG_LENGTH and IV_LENGTH only, using BITS_IN_BYTE multiplier where it is required (I have found only 2 places in our code).